frontend/src/Components/CustomNavbar.tsx

```diff
@@ -8,7 +8,10 @@ import { useNavigate } from 'react-router-dom';
 
 
 function CustomNavbar() {
-  const [permission, editPermission] = useState<String>("Employee");
+  if (null == localStorage.getItem("Permission")) {
+    localStorage.setItem("Permission", "Employee")
+  }
+  const [permission, editPermission] = useState<String>(localStorage.getItem("Permission") || "Employee");
```

Comment:
is || "Employee" necessary since Permission should never be null at this point bc of line 11 and 12?

Reply:
Yes it is as localStorage.getItem returns String or Null datatype. Even if logically it will not be Null, conceptually it can be